biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -104,10 +113,17 @@ subroutine IsItLoggingTime(is_master,currentSite)
       logging_time = .false.
       icode = int(logging_event_code)
 
+      ! this is true for either hlm harvest or fates logging
       if(hlm_use_logging.eq.ifalse) return
 
+      ! all of these are valid for hlm harvest
+      ! so adjust annual harvest inputs accordingly in LoggingMortality_frac
+      ! note that the specific event will allow only one hlm harvest, regardless of input
+      ! code 3, every day, may not work properly because of the exclusive mortality selection
+      ! even less fequent low harvest rates may be excluded - may need to give harvest priority
+
```

Comment:
Are they valid in the sense that logging happens in addition to the harvesting? (with the caveat that the model has to choose one of them?) Would it make sense to have a harvesting operation AND a logging operation happening in the same simulation?

Reply:
This means that they can each be set as an input value, and they should run.
The issue is that if the harvest is distributed in time it may be superseded by another disturbance on any given day because the largest disturbance wins the day (which would reduce the total annual harvest simulated).